src/main/java/com/denizenscript/clientizen/objects/LocationTag.java

```diff
@@ -18,6 +18,31 @@
 
 public class LocationTag implements ObjectTag, VectorObject {
 
+    // <--[ObjectType]
+    // @name LocationTag
+    // @prefix l
+    // @base ElementTag
+    // @implements VectorObject
+    // @ExampleTagBase client.cursor_on
+    // @ExampleValues <client.cursor_on>
+    // @ExampleForReturns
+    // - narrate "The location is %VALUE%!"
+    // @format
+    // The identity format for locations is <x>,<y>,<z>,<pitch>,<yaw>,<world>
+    // Note that you can leave off the world, and/or pitch and yaw, and/or the z value.
```

Comment:
and/or not needed here
"Note that you can **omit** the world value. You can also omit **either** the pitch and yaw pair or the z value."